README.md

```diff
@@ -1483,6 +1483,7 @@ Table of Contents
    * [PostHog](https://posthog.com) - Full Product Analytics suite free for up to 1m tracked events per month
    * [Uptrace](https://uptrace.dev) - Distributed Tracing Tool that helps developers pinpoint failures and find performance bottlenecks. Has a free plan, offers a free Personal subscription for open source projects, and has an open source version.
    * [Microsoft Clarity](https://clarity.microsoft.com/) - Clarity is a free, easy-to-use tool that captures how real people actually use your site.
+   * [Alceris Analytics](https://alceris.com) - Alceris Analytics is a privacy focused website analytics solution which creates insights without using any user identifing data or cookies. EU hosted and GDPR compliant. Free plan includes 1000 tracked events and pageviews per month.
```

Comment:
a 1000 page views a month? This is not useful.

Reply:
The free package is intended for smaller sites and people which want to test the functionality. But if it's not a good fit for the list I can close the pull request.

Reply:
Yeah I think so, we expect people with trials to allow a year trial for example so it has to be significant enough to do useful work, a 1000 hits, thats not a lot even for local dev imo